src/gui/threejsScene.js

```diff
@@ -78,6 +79,12 @@ import { InfiniteGridHelper } from '../../thirdPartyCode/THREE.InfiniteGridHelpe
 
         addGroundPlaneCollider(); // invisible object for raycasting intersections with ground plane
 
+        let pmremGenerator = new THREE.PMREMGenerator(renderer);
+        pmremGenerator.compileEquirectangularShader();
+
+        let neutralEnvironment = pmremGenerator.fromScene(new RoomEnvironment()).texture;
+        scene.environment = neutralEnvironment;
```

Comment:
Is this analogous to adding an HDRI to the scene for lighting?

Reply:
Yep! (had to look up what an HDRI is 🤣 )